tests/SourcemapToolkit.CallstackDeminifier.UnitTests/FunctionMapGeneratorUnitTests.cs

```diff
@@ -390,36 +405,39 @@ public void ParseSourceCode_InstanceMethodWithObjectInitializerAndFunctionHasNam
 		}
 
 		[Fact]
-		public void GetDeminifiedMethodNameFromSourceMap_NullFunctionMapEntry_ThrowsException()
+		public void GetDeminifiedMethodNameFromSourceMap_NullBindings_ReturnsNull()
 		{
 			// Arrange
-			FunctionMapEntry functionMapEntry = null;
-			SourceMap sourceMap = MockRepository.GenerateStub<SourceMap>();
+			IReadOnlyList<BindingInformation> bindings = null;
+			SourceMap sourceMap = CreateSourceMapMock();
 
 			// Act
-			Assert.Throws<ArgumentNullException>( ()=> FunctionMapGenerator.GetDeminifiedMethodNameFromSourceMap(functionMapEntry, sourceMap));
+			string deminifiedMethodName = SourceMapExtensions.GetDeminifiedMethodName(sourceMap, bindings);
+
+			// Assert
+			Assert.Null(deminifiedMethodName);
 		}
 
 		[Fact]
 		public void GetDeminifiedMethodNameFromSourceMap_NullSourceMap_ThrowsException()
```

Comment:
Looks like this is no longer the unit being tested. Could you update the names of these tests please? I think this applies to a few refactors here.

Reply:
I got rid of the "FromSourceMap" from the name when I moved it to be an extension method, but the feedback is fair.  I just sent an update to reflect that.